src/Controller/HomepageController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Entity\DoStuff;
+use App\Util\Debugger;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class HomepageController
+{
+    public function __invoke(Request $request, Container $container)
+    {
+        switch ($request->getMethod()) {
+            case "GET":
+                return $this->index($request, $container);
+                break;
+            default:
+                throw new Exception("verb non supporté");
+        }
+    }
+
+    /**
+     * GET HomepageController
+     */
+    public function index(Request $request, Container $container)
```

Comment:
Meh Using a different controller would have been better. 
1 class 1 purpose.
It's better for multiple reasons, testing, maintaining, readability, reduce the amount of data to load and interpret, etc...